notebooks/official/feature_store/online_feature_serving_and_vector_retrieval_bigquery_data_with_feature_store.ipynb

```diff
@@ -88,20 +88,6 @@
         "- Use the online server to search nearest neighbors."
```

Comment:
<p>nit: no need to change, as embedding is staying in Preview until Q1 anyways.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>